docs/using-openstack-cloud-controller-manager.md

```diff
@@ -49,7 +49,7 @@ The following guide has been tested to install Kubernetes v1.17 on Ubuntu 18.04.
     kubectl create secret -n kube-system generic cloud-config --from-file=cloud.conf
     ```
 
-- Create RBAC resources and openstack-cloud-controller-manager deamonset.
+- Create RBAC resources and openstack-cloud-controller-manager deamonset. Optionaly if you use certs then put the cert file into folder `/etc/ssl/certs/`, and update `ca-file` in configuration file `cloud.conf` above accordingly.
```

Comment:
It's a little bit late to mention 'updating ca-file' here after the config secret already being created, we could reference https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-openstack-cloud-controller-manager.md#global in step 4 and add more description for the option `ca-file` e.g. 

```
`ca-file` Optional. CA certificate bundle file for communication with Keystone service, this is required when using the https protocol in the Keystone service URL. openstack-cloud-controller-manager should have access to the file location.
```

Reply:
ok, I will see what I can do 
at least, my env is https so I think it's helpful to folks who use https env :)